admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -69,7 +68,7 @@ export class AlertBannerModel extends HoistModel {
     }
 
     get supportPresets() {
-        return checkMinVersion(XH.getEnv('hoistCoreVersion'), '16.3.0');
+        return XH.environmentService.isMinHoistCoreVersion('16.3.0');
```

Comment:
Woohoo - saved us two chars! 😉 

We could alias off of `XH` if we wanted - I appreciate the clarity of the long method name, though